README.md

```diff
@@ -121,14 +122,17 @@ Source code heuristics:
 | shady-links | Identify when a package contains an URL to a domain with a suspicious extension |
 | npm-exec-base64 | Identify when a package dynamically executes code through 'eval' |
 | npm-install-script | Identify when a package has a pre or post-install script automatically running commands |
+| npm-exfiltrate-sensitive-data | Identify when a package reads and exfiltrates sensitive data from the local system |
+| npm-exfiltrate-sensitive-data | Identify when a package reads and exfiltrates sensitive data from the local system |
```

Comment:
duplicate?

Reply:
It seems a bug with the doc generator when a rule has two languages belonging to the same ecosystem. e.g: javascript, typescript
I'll fix it

Reply:
fixed in https://github.com/DataDog/guarddog/pull/346/commits/d10a646a725f3ccb6d66a5c9e5dce7326e1b542d